WORKSPACE

```diff
@@ -0,0 +1,85 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:jvm.bzl", "jvm_maven_import_external")
+
+android_sdk_repository(
```

Comment:
Can you stack this PR so that it is diffed off of a branch that introduces the WORKSPACE so that it doesn't need to be introduced in each file?

Reply:
Done :)

Reply:
Thanks!